bundle-size/cloud_build.yaml

```diff
@@ -0,0 +1,46 @@
+steps:
+  # The first two steps are required to identify and inject un-redacted secrets.
+  - name: gcr.io/cloud-builders/npm
+    args:
+      - install
+      - yaml
+  - name: gcr.io/cloud-builders/npm
+    args:
+      - run
+      - replace-secrets
+      - --
+      - bundle-size
+    secretEnv:
+      [
+        'WEBHOOK_SECRET',
+        'PRIVATE_KEY',
+        'DATABASE_CONNECTION_JSON',
+        'TRAVIS_PUSH_BUILD_TOKEN',
+        'ACCESS_TOKEN',
+      ]
+
+  # Install app dependencies
+  - name: gcr.io/cloud-builders/npm
+    dir: bundle-size
+    args:
+      - install
+
+  # Deploy the app and its cron tasks
+  - name: gcr.io/cloud-builders/gcloud
+    dir: bundle-size
+    args:
+      - --project
+      - amp-bundle-size-bot
```

Comment:
This `--prooject amp-bundle-size-bot` bit shouldn't be here. I know I included it for `bundle-size-chart`, but I shouldn't have and it should be removed (it's just there because it was in the `deploy` command in `package.json`. These commands are running within the GAE project instance, so there's no need or point to including this parameter here, in `bundle-size-chart`, or in `test-status`

Reply:
Done.